scripts/smoke-checks/Readme.md

```diff
@@ -2,5 +2,5 @@
 
     With the local fork (Anvil):
     ```
-    forge script scripts/smoke-checks/DeployHoleskyMocksLidoRolesValidator.s.sol:DeployHoleskyMocksLidoRolesValidator --fork-url http://localhost:8545 --broadcast --account Deployer1 --sender <DEPLOYER1_ADDRESS> --verify
+    forge script scripts/smoke-checks/holesky-dry-run/DeployHoleskyMocksLidoRolesValidator.s.sol:DeployHoleskyMocksLidoRolesValidator --fork-url http://localhost:8545 --broadcast --account Deployer1 --sender <DEPLOYER1_ADDRESS> --verify
```

Comment:
`--sender` flag overrides `--account` and as far as they are the same its enough to specify only account
https://github.com/foundry-rs/foundry/pull/7141